backend/src/packages/chaiNNer_standard/image_utility/miscellaneous/change_colormodel.py

```diff
@@ -26,10 +26,10 @@
 
 
 @miscellaneous_group.register(
-    schema_id="chainner:image:change_colorspace",
-    name="Change Colorspace",
+    schema_id="chainner:image:change_colormodel",
```

Comment:
You can't just change a schema id without also creating a migration. Changing this will break all existing chains that use this node.

Reply:
Also, there's no need to change the schema ID, because it is purely internal.

Reply:
left Schema ID as is